src/APIFunctions/Mailer.js

```diff
@@ -14,9 +14,38 @@ export async function sendVerificationEmail(email, firstName) {
   await axios
     .post(MAILER_API_URL + '/api/Mailer/sendVerificationEmail', {
       recipientEmail: email,
-      recipientName: firstName
+      recipientName: firstName,
     })
-    .catch(error => {
+    .then((response) => {
+      status.responseData = response;
+    })
+    .catch((error) => {
+      status.error = true;
+      status.responseData = error;
+    });
+  return status;
+}
+
+/**
+ * Invoke the gmail API to send an email blast to specified users.
+ * @param {Array} emailList - String array of user email addresses
+ * @param {string} subject - The subject of the email
+ * @param {string} content - The contents of the email
+ * @returns {ApiResponse} Containing any error information related to the
+ * request
+ */
+export async function sendBlastEmail(emailList, subject, content) {
+  let status = new ApiResponse();
+  await axios
+    .post(MAILER_API_URL + '/api/Mailer/sendBlastEmail', {
+      recipientEmail: emailList,
```

Comment:
if it possible you chould change the backend to accept
`req.body.emailList, req.body.subject, req.body.content`?

that way you can do 
```js
.post('/path/to/api', {
  emailList,
  subject,
  content
})
```

Reply:
Got it, lmk if you want me to push the changes

Reply:
yes please